### ISSUE TITLE
Wools dont re-appear in chests

### ISSUE THREAD

**Issue by TiTo-418** [twizmwazin/CardinalPGM#431]

**Comment by AngelCasillas01** [comment]
This problem is kind of the same, but the difference is that this issue is with phisical blocks instead of inventory blocks

#434

**Comment by TiTo-418** [comment]
@AngelCasillas01 ok, agrega esta issue a la #434 y cierro esta

**Comment by CaptainElliott** [comment]
This is a different feature

**Comment by AngelCasillas01** [comment]
@TiTo-418 no dejala abiera para saber cuando lo resuelvan xaval

**Comment by Alfalik** [comment]
Auto-refill chest is only able on Public OCN. In tourneys it's disabled to prevent safety spam. I don't think that this should be added

**Comment by TiTi-418** [comment]
@Alfalik If you take all the wools?

**Comment by Alfalik** [comment]
@TiTi-418 Yes. If you pick up all the wools in the chest, you easily can spam safeties when bringing it back, wich make an easier job for your team. Disabling auto-refill prevent it.

**Comment by TiTi-418** [comment]
If you take all the wools, and lose it, you can't win the match

**Comment by Alfalik** [comment]
@TiTi-418 The chest have 54 wools, I don't see why do you need to pick up all. Auto-refill is a new feature that hasn't been used in old tourneys, and actually isn't used.

**Comment by TiTi-418** [comment]
If i arrive to the wool room 55 times, i can't get one. If i arrive 6 times to the wool room and i pick up 14 wools, i can't get one etc,etc

**Comment by Superjd10** [comment]
Tourney maps are made to be strategic. What would be the point of coming to a wool room, take all the wools, block spam it and if you die, go back to the wool room and repeat? 

If it's disabled in the Tourney Plugin or module of PGM, it should be disabled / not added in Cardinal.

**Comment by TiTi-418** [comment]
Maybe a toggeable option?

**Comment by twizmwazin** [comment]
What I am currently working with is a configurable rate (mostly because I don't know the actual rate). By default it will be set to 0, so disabled.

**Comment by RubeGamerV** [comment]
This can be a module and be toggleable in the map xml

**Comment by felipufeliu** [comment]
@titi-418 Then if you want to get 55 times and get 55 times out of the wool and die try to organize better before going out. Also auto refill haven't been used in any Overcast Network tournament so far so at this point is not needed.

**Comment by Pablete1234** [comment]
This is only used for public, in TM and in Tournaments it's disabled completly, should this really be an issue?

**Comment by twizmwazin** [comment]
Yes, because this is a feature that should some day be implemented.
